### PR TITLE
Fix authaddress determined in nodeserver

### PR DESCRIPTION
**Pull request by ajainarayanan**
Unit test: https://builds.cask.co/browse/CDAP-UDUT395-1
E2E test: https://builds.cask.co/browse/IT-UPD272-1

### CODE REVIEW

**Comment by ajainarayanan** [comment]
Consecutive 6 Mini builds ,
[Build 1](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=16&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-21),  [Build 2](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=20&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-20), [Build 3](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=20&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-19), [Build 4](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=20&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-18), [Build 5](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=20&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-17), [Build 6](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI6&buildNumber=20&returnUrl=%2Fbrowse%2FIT-UIE2EMINI6-16)

Consecutive 4 E2E tests [E2E Build 1](https://builds.cask.co/browse/IT-UPD272-17), [E2E Build 2](https://builds.cask.co/browse/IT-UPD272-16), [E2E Build 3](https://builds.cask.co/browse/IT-UPD272-15), [E2E Build 4](https://builds.cask.co/browse/IT-UPD272-14)

**Review by yukiej — APPROVED** [review]
lgtm!

File: cdap-ui/cypress/integration/login.spec.ts [hunk 1]
```diff
@@ -45,6 +47,7 @@ describe('Logging in', () => {
       .type(INCORRECT_LOGIN);
     cy.get('#submit').click();
     cy.url().should('include', '/login');
+    console.log(cy.getCookie('CDAP_Auth_Token'));
```
Comment (elfenheart) [inline cdap-ui/cypress/integration/login.spec.ts]: I don't think we should console out the auth token here

Comment (ajainarayanan) [reply cdap-ui/cypress/integration/login.spec.ts]: ah! Dev change. Will revert back.

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**